src/main/java/uk/gov/justice/probation/courtcaseservice/service/model/UnpaidWork.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.probation.courtcaseservice.service.model;
+
+import io.swagger.annotations.ApiModel;
+import lombok.Builder;
+import lombok.Getter;
+
+@ApiModel("UnpaidWork")
+@Getter
+@Builder
+public class UnpaidWork {
+    private Integer offered;
```

Comment:
We should document that this value is given in minutes... in fact maybe just rename it to minutesOffered and not bother with documenting it. Sorry I appreciate this is what it says in the ticket!

Reply:
I agree, I did think that when I was doing it actually. I'll update it.

Reply:
Awesome! Don't feel like the ticket is gospel, that was just my best interpretation of what seemed sensible at the time but we should feel free to change it if we find things that can be done better. Just leave a comment or something so people coming after know.